app/controllers/concerns/search_aggregator.rb

```diff
@@ -24,39 +24,39 @@ module SearchAggregator
     /semanticweb/i
   ]
 
-  def aggregate_results(query, results)
+  def aggregate_results(query, results, lang)
```

Comment:
to not pass `lang` as parameter on each function, you can use the global helper `helpers.request_lang`